core/tests/epoch_accounts_hash.rs

```diff
@@ -90,13 +89,15 @@ impl TestEnvironment {
 
     #[must_use]
     fn _new(snapshot_config: SnapshotConfig) -> TestEnvironment {
+        const MINT_LAMPORTS: u64 = 100_000 * LAMPORTS_PER_SOL;
+        const STAKE_LAMPORTS: u64 = 100 * LAMPORTS_PER_SOL;
```

Comment:
I did not know this, but the compiler will evaluate this at compile time and fail to compile if a constant would overflow. Pretty neat!